ang/test/civicase/ActivitiesCalendar.spec.js

```diff
@@ -0,0 +1,261 @@
+/* eslint-env jasmine */
+
+(function ($, _, moment) {
+  describe('civicaseActivitiesCalendarController', function () {
+    var $controller, $scope, $rootScope, activitiesCalendar, activitiesMockData,
+      dates, formatActivity, mockCaseId;
+
+    beforeEach(module('civicase', 'civicase.data'));
+
+    beforeEach(inject(function (_$controller_, _$rootScope_,
+      _activitiesMockData_, datesMockData, _formatActivity_) {
+      $controller = _$controller_;
+      $rootScope = _$rootScope_;
+      activitiesMockData = _activitiesMockData_.get();
+      dates = datesMockData;
+      formatActivity = _formatActivity_;
+    }));
+
+    describe('calendar options', function () {
+      beforeEach(function () {
+        initController();
+      });
+
+      it('hides the weeks panel from the calendar', function () {
+        expect(activitiesCalendar.calendarOptions).toEqual(jasmine.objectContaining({
+          showWeeks: false
+        }));
+      });
+    });
+
+    describe('selected activities', function () {
+      beforeEach(function () {
+        initController();
+      });
+
+      describe('when selecting a date with activities included', function () {
+        var expectedSelectedActivities;
+
+        beforeEach(function () {
+          spyOn($scope, '$emit').and.callThrough();
+
+          expectedSelectedActivities = activitiesMockData
+            .filter(function (activity) {
+              return moment(activity.activity_date_time).isSame(dates.today, 'day');
+            })
+            .map(function (activity) {
+              return formatActivity(activity, mockCaseId);
+            });
+
+          activitiesCalendar.selectedDate = dates.today;
+          activitiesCalendar.onDateSelected();
+        });
+
+        it('only provides the activities for the given date', function () {
+          expect(activitiesCalendar.selectedActivites).toEqual(expectedSelectedActivities);
+        });
+
+        it('emits an "open activities popover" event', function () {
```

Comment:
I think unit tests description should be as much non technical as possible.
`it('opens activities popover'`